databricks/sdk/errors/mapper.py

```diff
@@ -1,8 +1,72 @@
 from databricks.sdk.errors import platform
 from databricks.sdk.errors.base import DatabricksError
+from dataclasses import dataclass
+import re
+from typing import Optional
+import requests
+from .platform import ResourceDoesNotExist
 
 
-def error_mapper(status_code: int, raw: dict) -> DatabricksError:
+@dataclass
+class _ErrorOverride:
+    # The name of the override. Used for logging purposes.
+    debug_name: str
+
+    # A regex that must match the path of the request for this override to be applied.
+    path_regex: re.Pattern
+
+    # The HTTP method of the request for the override to apply
+    verb: str
+
+    # The custom error class to use for this override.
+    error_class: type
+
+    # A regular expression that must match the error code for this override to be applied. If None,
+    # this field is ignored.
+    error_code_regex: Optional[re.Pattern] = None
+
+    # A regular expression that must match the message for this override to be applied. If None,
+    # this field is ignored.
+    message_regex: Optional[re.Pattern] = None
+
+    def matches(self, response: requests.Response, raw_error: dict):
+        if response.request.method != self.verb:
+            return False
+        if not self.path_regex.match(response.request.path_url):
+            return False
+        if self.error_code_regex and not self.error_code_regex.match(raw_error.get('error_code', '')):
+            return False
+        if self.message_regex and not self.message_regex.match(raw_error.get('message', '')):
+            return False
+        return True
+
+
+_INVALID_PARAMETER_VALUE = 'INVALID_PARAMETER_VALUE'
+
+_all_overrides = [
```

Comment:
@mgyucht why it is not consistent with https://github.com/databricks/databricks-sdk-go/pull/682? why is it not inhttps://github.com/databricks/databricks-sdk-go/blob/main/openapi/errors.go first? it'll be inconsistent among other languages otherwise.

Reply:
Addressed this now.